pkgs/by-name/op/openbao/package.nix

```diff
@@ -0,0 +1,75 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+  go_1_24,
+  testers,
+  openbao,
+  versionCheckHook,
+  nix-update-script,
+}:
+
+buildGoModule.override { go = go_1_24; } rec {
```

Comment:
buildGo124Module is the correct alias to use here

Reply:
Good to Know